multiapps-controller-persistence/src/main/resources/org/cloudfoundry/multiapps/controller/persistence/db/changelog/db-changelog-1.116.0-persistence.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.6.xsd">
+
+    <changeSet author="sap.com" id="alter_table_lm_sl_persistence_file">
+        <preConditions onFail="MARK_RAN">
```

Comment:
The `db-changelog-1.110.0-xxx.xml` files are full of preconditions, because they replace previous Liquibase changelogs and they must be skipped in case their previous versions were already executed. However, this is a brand new change and it should always be executed, so that the `lm_sl_persistence_file` table is in a consistent state across all landscapes.

Remove this precondition and let the change fail if there are any problems.